src/common/regions/types.ts

```diff
@@ -22,6 +22,15 @@ const fipsToPrincipalCityRenames: FipsToPrincipalCityName = {
   [NY_METRO_FIPS]: 'New York City',
 };
 
+// PODT Region object to aid serialization/deserialization w/ JSON
+export interface RegionObject {
+  n: string;
+  u: string;
+  f: FipsCode;
+  p: number;
+  t: RegionType;
```

Comment:
Replied to the previous comment thread, but adding it again here just in case that gets lost.  I think we don't need to include `t` on the `RegionType` or write it in `toObject()`. We're not using it anywhere.

Reply:
True.  I had thought of this, but was leaving it in, because if we end up breaking out individual files for location pages we're going to need to be able to inspect the object itself to figure out what kind of region it is.  But, perhaps IAGNI, and also, we can write region type to those files specifically and just inspect it to hydrate the right one, so we don't need it in the files with everything and so I'll remove it and regenerate the data,